### PR TITLE
Provided a solution using Python 3 to phone number letters combinations algorithm.

### PR DESCRIPTION
**Pull request by yvantcheuffa**
- Provided a solution using Python 3 to phone number letters combinations algorithm.
- Added link to the solution in README.md file.

### CODE REVIEW

**Comment by dassiorleando** [comment]
Good thing @yvantcheuffa, Thank you.

Everything is alright for me but just to keep things centralized, could you consider the following?

- Instead of having two lines for the same challenge in the readme, we can use the first one and describe both solutions in the same file.
- Update the phone readme to highlight the first solution with its runtime (45ms), description and code and the second solution with your runtime (38 ms), description and code as well.
- The by line could have two names: foka and you.

What’s your take on that?

**Comment by yvantcheuffa** [comment]
> Good thing @yvantcheuffa, Thank you.
> 
> Everything is alright for me but just to keep things centralized, could you consider the following?
> 
> * Instead of having two lines for the same challenge in the parent readme, we can use the existing one and describe both solutions in the same child readme.
> * Update the phone readme(letter-combinations-phone-number.md) to highlight the first solution with its runtime (45ms), description and code and the second solution with your runtime (38 ms), description and code as well.
> * The “by line” could have two names: foka and you.
> 
> What’s your take on that?



> Good thing @yvantcheuffa, Thank you.
> 
> Everything is alright for me but just to keep things centralized, could you consider the following?
> 
> * Instead of having two lines for the same challenge in the parent readme, we can use the existing one and describe both solutions in the same child readme.
> * Update the phone readme(letter-combinations-phone-number.md) to highlight the first solution with its runtime (45ms), description and code and the second solution with your runtime (38 ms), description and code as well.
> * The “by line” could have two names: foka and you.
> 
> What’s your take on that?

I understand your point. Let me update this PR.

File: algorithms/letter-combinations-phone-number.md [hunk 1]
```diff
@@ -33,5 +33,49 @@ function letterCombination(digits: string): string[] {
     return !digits.length? [] : combinaison(digits, ['']);
 };
 ```
+#### Runtime: 45ms
+By [Noel Foka](https://github.com/noelfoka)
 
-By Noel Foka
+**Solution 2 (using Python 3)**: 
+Our solution is quite simple. First, we validate user's input and display appropriate error messages in case there is any error.
+Next, we create an empty list that will serve as an accumulator and another list for matches containing strings who values are the equivalent
```
Comment (messaismael) [inline algorithms/letter-combinations-phone-number.md:41]: ... strings `whose` values ...

File: algorithms/letter-combinations-phone-number.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ This challenge comes from Leetcode, it's about finding the number of zero-filled
 
 **Challenge**: https://leetcode.com/problems/letter-combinations-of-a-phone-number/.
 
-**Solution**: 
+**Solution 1 (using JavaScript)**:
```
Comment (messaismael) [inline algorithms/letter-combinations-phone-number.md:6]: The solution is in `TypeScript` rather

**Comment by messaismael** [comment]
@yvantcheuffa can you remove this file cause it's a duplication?

`letter-combinations-of-a-phone-number.md`

**Review by messaismael — CHANGES_REQUESTED** [review]
Can you remove this file also it's a duplication.

`letter-combinations-of-a-phone-number.md`

File: algorithms/letter-combinations-phone-number.md (hunk 2 above)
Comment (yvantcheuffa) [reply algorithms/letter-combinations-phone-number.md]: @messaismael Done !

**Review by dassiorleando — APPROVED** [review]

**Comment by dassiorleando** [comment]
PR merged, thanks for the efforts guys.

**Pull request merged**